plugins/home/src/api/CoreStorageVisitsApi.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { IdentityApi, StorageApi } from '@backstage/core-plugin-api';
+import { Visit } from './VisitsApi';
+import { VisitsApiFactory } from './VisitsApiFactory';
+
+/** @public */
+export type CoreStorageVisitsApiOptions = {
+  storageApi: StorageApi;
+  limit?: number;
+  identityApi: IdentityApi;
+};
+
+/**
+ * @public
+ * This is an implementation of VisitsApi that relies on a StorageApi
+ */
+export class CoreStorageVisitsApi extends VisitsApiFactory {
```

Comment:
How about we rename this to VisitsStorageApi instead?

Reply:
addressed in 29662110254b1e8be15dc7a739eddf005f0d9f5a